CMakeLists.txt

```diff
@@ -368,9 +368,13 @@ if(WITH_CORE)
   set (WITH_INTERNAL_NLOHMANN_JSON ON CACHE BOOL "Determines whether the vendored copy of nlohmann-json should be used")
   find_package(nlohmann_json REQUIRED)
 
+  # This bypasses the FindSQLite3 module introduced in CMake 3.14
+  # On case insensitive platforms (e.g. Windows) this is because
+  # ./cmake/FindSqlite3.cmake comes first on the CMAKE_MODULE_PATH
+  # (otherwise it is because of the case: *Sqlite3* vs. *SQLite3*)
   find_package(Sqlite3)
```

Comment:
I think we should just make it `REQUIRED` here and kill the checks below

Reply:
Fine for me I just kept it like it was before. :-)